core/src/main/java/com/bettercloud/bigtable/orm/BigTableEntityDao.java

```diff
@@ -127,10 +143,21 @@
                 bytes = null;
             }
 
-            put.addColumn(Bytes.toBytes(column.getFamily()), Bytes.toBytes(column.getQualifier()), bytes);
+            if (column.isVersioned()) {
+                final long timestamp = Optional.ofNullable(sourceDelegate.getColumnTimestamp(column))
+                        .orElseGet(() -> Instant.now().toEpochMilli());
```

Comment:
Nitpick: can `Instant.now().toEpochMilli()` be pulled into a constant? Also does it make sense to add to specify in the readme that this is what will be used as the default if a timestamp is not provided?

I guess it should be fairly obvious so take this for the nitpick that it is :)

Reply:
Can't really pull it into a constant, since it needs to be evaluated on-save.

Also, from the readme:

> Invoking the setter for a column without defining the timestamp parameter will set the timestamp to `null` on the Java Object. A `null` timestamp is interpreted as a signal to auto-generate the column's timestamp on `save`

Reply:
🤦‍♂️ I'm dumb. Yea I guess I was reaching a bit here for something. lgtm 👍